src/styles/_typography.scss

```diff
@@ -0,0 +1,31 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap');
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+.overline-text,
+.lead-text,
+.major-text {
+  font-weight: 500;
+  font-style: normal;
+}
+
+.overline-text {
+  font-size: 9px;
+  line-height: 12px;
```

Comment:
`line-height` powinno być ustawiane bez jednostek. Czyli w tym przypadku `line-height: 1.333;` czyli `12px / 9px`.
Dzięki temu zwiększając `line-height` np. na większym breakpointcie nie dopuścimy do sytuacji, że linie na siebie nachodzą bo `line-height` jest mniejszy niż `font-size`.

- https://allthingssmitty.com/2017/01/30/nope-nope-nope-line-height-is-unitless/
- https://www.paulingraham.com/trouble-with-unitless-line-heights.html